etc/async-aurora-test/src/engine_types.rs

```diff
@@ -0,0 +1,53 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+
+pub type EthAddress = [u8; 20];
+pub type RawU256 = [u8; 32];
+/// Wei compatible Borsh-encoded raw value to attach an ETH balance to the transaction
+pub type WeiU256 = [u8; 32];
+
```

Comment:
NB: this file is being replicated twice in both `etc/async-aurora-test` and `etc/async-aurora`. Moreover, it shouldn't be duplicated there, rather should be separated into a separate lib in the repo itself.